corpora/proverbs.dep

```diff
@@ -0,0 +1,1826 @@
+# text = Айды этек менен калкалай албасың.
+# text[eng] = You cannot shield the moon with the bottom hem of a shirt.
+# labels = JNW_checked_cg JNW_initialcheck_dep
+"<Айды>"
+	"ай" n acc @obj #1->4
+"<этек>"
+	"этек" n nom @obl #2->4
+"<менен>"
+	"менен" post @case #3->2
+"<калкалай>"
+	"калкала" v tv prc_impf @root #4->0
+"<албасың>"
+	"ал" vaux neg aor p2 sg @aux #5->4
+"<.>"
+	"." sent @punct #6->4
+
+# sent_id = proverbs:47
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:282
+# text = Колунда токочу бар бала сүйгүнчүктүү.
+# text[eng] = The child with a loaf (of bread) in their hand is darling.
+# labels = JNW_checked_cg tocheck_dep
+"<Колунда>"
+	"кол" n px3sp loc @obl #1->2
+"<токочу>"
+	"токоч" n px3sp nom @nmod:poss#2->4
+"<бар>"
+	"бар" adj @amod #3->4
+"<бала>"
+	"бала" n nom @nsubj #4->6
+"<сүйгүнчүктүү>"
+	"сүйгүнчүктүү" adj @root #5->0
+		"э" cop aor p3 sg @cop #6->5
+"<.>"
+	"." sent @punct #7->5
+
+# sent_id = proverbs:48
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:153
+# text = Жазмыштан озуш жок.
+# text[eng] = There is no getting ahead of destiny.
+# labels = JNW_checked_cg tocheck_dep
+"<Жазмыштан>"
+	"жазмыш" n abl @obl #1->2
+"<озуш>"
+	"оз" v iv ger_pres nom @nsubj #2->3
+"<жок>"
+	"жок" adj @root #3->0
+		"э" cop aor p3 sg @cop #4->3
+"<.>"
+	"." sent @punct #5->3
+
+# sent_id = proverbs:49
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:134
+# text = Бузулган элгe бучуктан молдо болот.
+# text[eng] = The mullah of a rotten people is from among the noseless/flat-nosed.
+# labels = JNW_checked_cg tocheck_dep
+"<Бузулган>"
+	"буз" v tv pass gpr_past @amod #1->2
+"<элгe>"
+	"эл" n dat @obl #2->5
+"<бучуктан>"
+	"бучук" adj subst abl @obl #3->4
+"<молдо>"
+	"молдо" n nom @nsubj #4->5
+"<болот>"
+	"бол" v iv aor p3 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:50
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:53
+# text = Ар кимде бар бир кыял, ар кыялды ким тыяр.
+# text[eng] = Everyone has a personality, no one can stop any personality.
+# labels = JNW_checked_cg tocheck_dep
+"<Ар кимде>"
+	"ар ким" prn ind loc @obl #1->2
+"<бар>"
+	"бар" adj @root #2->0
+		"э" cop @cop #3->2
+"<бир>"
+	"бир" det ind @det #4->5
+"<кыял>"
+	"кыял" n nom @nsubj #5->2
+"<,>"
+	"," cm @punct #6->10
+"<ар>"
+	"ар" det ind @det #7->8
+"<кыялды>"
+	"кыял" n acc @obj #8->10
+"<ким>"
+	"ким" prn itg nom @nsubj #9->10
+"<тыяр>"
+	"тый" v iv gpr_fut @parataxis #10->2
+"<.>"
+	"." sent @punct #11->10
+
+# sent_id = proverbs:51
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:52
+# text = Ар ким ич оорусун өзү билет.
+# text[eng] = Everyone knows their stomache ache themselves.
+# labels = JNW_checked_cg tocheck_dep
+"<Ар ким>"
+	"ар ким" prn ind nom @nsubj #1->5
+"<ич>"
+	"ич" n nom @nmod:poss #2->3
+"<оорусун>"
+	"оору" n px3sp acc @obj #3->5
+"<өзү>"
+	"өз" prn ref px3sg nom @nmod:poss #4->1
```

Comment:
Probably either `@appos #4->1` or `@advmod #4->5`.  I'm currently leaning towards the latter.

Reply:
Open question for discussion!